ext/webidl/00_webidl.js

```diff
@@ -698,7 +697,7 @@ function createDictionaryConverter(name, ...dictionaries) {
     }
     const esDict = V;
 
-    const idlDict = ObjectAssign({}, defaultValues);
+    const idlDict = { __proto__: null, ...defaultValues };
```

Comment:
I'm a bit queasy about the `__proto__` usage, given that Deno doesn't support it normally. eg. there's another PR open about how to handle NPM libraries that rely on `__proto__` and thus fail in odd ways when run in Deno through `npm:` imports.

Would it be an option to use `ObjectAssign(ObjectCreate(null), defaultValues)`?

Reply:
`__proto__` is something [defined in the ECMAScript spec](https://tc39.es/ecma262/#sec-runtime-semantics-propertydefinitionevaluation), and that file is written in JS, a language that conforms to the ECMAScript spec, so it doesn't really matter whether Deno supports it when running user code I think.

> Would it be an option to use `ObjectAssign(ObjectCreate(null), defaultValues)`?

It would be an option, but it typically has much worse performance on current versions of V8. IMHO it's also less readable.

Reply:
There are two spec for `__proto__` in ECMAScript

1. Set the `[[Prototype]]` of the object by using it in object literals
2. Change the `[[Prototype]]` of the object by `Object.prototype.__proto__` accessor property (normative optional, legacy)

The first one works fine in Deno.